src/panels/MissionsPanel/MissionContent.tsx

```diff
@@ -104,14 +118,14 @@ export function MissionContent({ missionOverview }: Props) {
   }
 
   return (
-    <Group wrap={'nowrap'} align={'start'} h={'100%'} gap={0}>
+    <Group wrap={'nowrap'} align={'start'} gap={'xs'}>
       <TimeLine
         allPhasesNested={allPhasesNested}
         displayedPhase={displayedPhase}
         missionOverview={missionOverview}
         setDisplayedPhase={setPhaseManually}
       />
-      <Box px={'md'} h={'100%'} style={{ overflow: 'auto' }}>
```

Comment:
I couldn't see any noticeable differences with the height and overflow css here, so I removed it (is it legacy from before @ylvaselling  refactor?) and moved the padding to the parent gap prop instead